client/src/App.css

```diff
@@ -313,7 +319,6 @@ html {
 .table-score {
   font-size: 16px;
   background-color: #ffffff;
-  /* margin-left: 15px; */
```

Comment:
looks good, however please delete the unnecessary linen

Reply:
sure

Reply:
the red colour means I've removed it